pages/tasks/index.js

```diff
@@ -58,29 +128,68 @@ const Sidebar = ({ filters, setFilters }) => {
 
 const TaskPage = ({ issues }) => {
   const { t } = useTranslation()
+  const { theme } = useTheme()
+  const isLight = theme === 'light'
   const [filters, setFilters] = useState({})
+  const [searchQuery, setSearchQuery] = useState('')
 
   const filteredIssues = issues.filter((issue) => {
     return !filters.status || issue.state === filters.status
   })
-
   return (
     <>
       <Head>
         <title> Tasks - WEB3DEV</title>
       </Head>
-      <div className="flex flex-col sm:flex-row">
-        <Sidebar filters={filters} setFilters={setFilters} t={t} />
-        <div className="flex-1 p-4">
-          {filteredIssues.length === 0 ? (
-            <p>{t('no-issues-found')}.</p>
-          ) : (
-            <div className="mt-4 grid grid-cols-1 gap-4 sm:grid-cols-2 lg:grid-cols-4">
-              {filteredIssues.map((issue) => (
-                <IssueCard key={issue.github_id} issue={issue} t={t} />
-              ))}
+      <div className="flex w-full items-center justify-center">
+        <div className="flex xl:w-[80%] w-full flex-col">
+          <SearchBar searchQuery={searchQuery} setSearchQuery={setSearchQuery} />
+          <div className="flex">
+            <div className="flex w-full flex-col items-start lg:flex-row md:mx-4">
+              <Filter />
+              <div className="flex-1 p-2 ">
+                {filteredIssues.length === 0 ? (
+                  <p>{t('no-issues-found')}.</p>
+                ) : (
+                  <div className="flex flex-col gap-2">
+                    <div className="flex h-10 flex-row items-center justify-between">
+                      <Sortbar filters={filters} setFilters={setFilters} t={t} />
+                      <label className={`h-10 w-[80px] md:w-[100px] md:text-[16px] text-[10px] ${
+                        isLight ? 'text-black-400' : 'text-[#99e24d]'}`}>247 PROJECT</label>
```

Comment:
This should not be hardcoded but a counter from the issues coming from db

Reply:
@nomadbitcoin, I have already adjusted it to display the number of Tasks that are filtered and that will be available to that user.